src/csharp/Microsoft.Spark/Microsoft.Spark.csproj

```diff
@@ -15,4 +27,18 @@
     <PackageReference Include="System.Memory" Version="4.5.2" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0-beta2-18618-05" PrivateAssets="All"/>
+  </ItemGroup>
+
+  <ItemGroup>
+    <Content Include="..\..\scala\microsoft-spark-*\target\microsoft-spark-*.jar" Link="jars\%(Filename)%(Extension)">
+      <Pack>true</Pack>
+      <PackagePath>jars\%(Filename)%(Extension)</PackagePath>
+    </Content>
+    <Content Include="..\build\**" Link="build\%(Filename)%(Extension)">
```

Comment:
What do you feel about moving this file into a `src/csharp/Microsoft.Spark/build/netstandard2.0` folder?

I have 2 concerns:

1. If we add other nupkgs that need build scripts, they can't go into this root `build` directory, or else they would start being packed here.
2. The .csproj shouldn't be deciding where this file goes. Instead the file system should.  If you put a file in `build/foo/bar/myFile.targets`, it should be in the same folder in the package. 

So instead of the above, you could have:

```xml
<Content Include="build\**\*" Pack="true" PackagePath="build" />
```

And it would pick up any file/folder structure under the `build` folder.

Reply:
I was trying to not move things around :) but agree it is better with this approach. Done!